src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -156,6 +156,16 @@ def track_initial_segment_event(
         return None
 
     if not project.flags.has_replays:
+        logger.info(
+            "Sending first_replay_received signal for project %d",
+            project_id,
```

Comment:
would just remove this, and only log project.id below. other than that looks good!

Reply:
No big problem with keeping it right? For redundancy i guess

Reply:
it would be nice to have project id as its own field, so we can filter on it. as it is now, wed have to do a full string match in order to get a particular project's log